install.sh

```diff
@@ -2842,34 +1972,75 @@ FIREEDGE_DEV_FILES="src/fireedge/src \
+                                src/fireedge/etc/sunstone/views/admin/vm-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vm-group-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/marketplace-app-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vnet-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vnet-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/image-tab.yaml\
+                                src/fireedge/etc/sunstone/views/admin/file-tab.yaml\
+                                src/fireedge/etc/sunstone/views/admin/sec-group-tab.yaml\
+                                src/fireedge/etc/sunstone/views/admin/backup-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/datastore-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vdc-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/user-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/service-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/service-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vrouter-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vrouter-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/backupjobs-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/host-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/group-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/acl-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/cluster-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/support-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/zone-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/marketplace-tab.yaml"
+
+FIREEDGE_SUNSTONE_ETC_VIEW_USER="src/fireedge/etc/sunstone/views/user/vm-tab.yaml \
+                                src/fireedge/etc/sunstone/views/user/vm-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/user/marketplace-app-tab.yaml \
+                                src/fireedge/etc/sunstone/views/user/image-tab.yaml\
+                                src/fireedge/etc/sunstone/views/user/file-tab.yaml\
+                                src/fireedge/etc/sunstone/views/user/backup-tab.yaml \
+                                src/fireedge/etc/sunstone/views/user/sec-group-tab.yaml \
+                                src/fireedge/etc/sunstone/views/user/vnet-tab.yaml"
+
+FIREEDGE_SUNSTONE_ETC_VIEW_GROUPADMIN="src/fireedge/etc/sunstone/views/groupadmin/vm-tab.yaml \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

FIREEDGE_SUNSTONE_ETC_VIEW_GROUPADMIN appears unused. Verify use (or export if used externally).